docs/README.md

```diff
@@ -0,0 +1,19 @@
+# Welcome to Win32 App Isolation Introduction
```

Comment:
I don't think "Welcome to" is very commonly used for documentation. Have you seen this elsewhere?